### ISSUE TITLE
Error with Campaign integration

### ISSUE THREAD

**Issue by AndrewMac** [verbb/formie#328]
**Description**
I've just noticed that I'm getting the following Craft Queue error on a form with Craft Campaign integration (but I'm afraid I don't know how long ago it started):

`API error: “Class 'verbb\formie\events\ModifyFieldValueForIntegrationEvent' not found” path/to/site/vendor/verbb/formie/src/elements/Submission.php:610`

The form used to be working fine, and I can't think of anything I might have changed recently that might give any kind of hint as to what is now going wrong.

The queued job is called:
"Triggering form “campaign” integration"

And this is the job data reported in the queue:

```
{
    "submissionId": 5374,
    "integration": {
        "listId": "4999",
        "optInField": "",
        "fieldMapping": {
            "email": "{email}",
            "firstName": "{name1[firstName]}",
            "lastName": "{name1[lastName]}",
            "nickname": "",
            "companyName": "{businessName}",
            "customerType": "{customerType}",
            "mailoutLanguage": "{mailoutLanguage}"
        },
        "name": "Campaign",
        "handle": "campaign",
        "type": null,
        "enabled": "1",
        "sortOrder": "4",
        "cache": {
            "settings": {
                "lists": [
                    {
                        "class": "verbb\\formie\\models\\IntegrationCollection",
                        "id": "4999",
                        "name": "... CAMPAIGN NAME ...",
                        "fields": [
                            {
                                "class": "verbb\\formie\\models\\IntegrationField",
                                "handle": "email",
                                "name": "Email",
                                "type": null,
                                "required": true,
                                "options": []
                            },
                            {
                                "class": "verbb\\formie\\models\\IntegrationField",
                                "handle": "firstName",
                                "name": "First Name",
                                "type": "string",
                                "required": false,
                                "options": []
                            },
                            {
                                "class": "verbb\\formie\\models\\IntegrationField",
                                "handle": "lastName",
                                "name": "Last Name",
                                "type": "string",
                                "required": false,
                                "options": []
                            },
                            {
                                "class": "verbb\\formie\\models\\IntegrationField",
                                "handle": "nickname",
                                "name": "Nickname",
                                "type": "string",
                                "required": false,
                                "options": []
                            },
                            {
                                "class": "verbb\\formie\\models\\IntegrationField",
                                "handle": "companyName",
                                "name": "Company Name",
                                "type": "string",
                                "required": false,
                                "options": []
                            },
                            {
                                "class": "verbb\\formie\\models\\IntegrationField",
                                "handle": "customerType",
                                "name": "Customer Type",
                                "type": "array",
                                "required": false,
                                "options": []
                            },
                            {
                                "class": "verbb\\formie\\models\\IntegrationField",
                                "handle": "mailoutLanguage",
                                "name": "Mailout Language",
                                "type": "array",
                                "required": false,
                                "options": []
                            }
                        ]
                    }
                ]
            }
        },
        "tokenId": null,
        "uid": "6269bd74-26c6-49cc-ad4c-79f576134aad",
        "referrer": "... OUR SITE URL ...",
        "id": "8",
        "dateCreated": "2020-10-14T03:13:06+11:00",
        "dateUpdated": "2020-11-03T16:39:34+11:00"
    },
    "description": null
}
```

**Steps to reproduce**
1.
2.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.16
- Craft version: Craft Pro 3.6.2
- Multi-site: Yes
- Craft Campaign version: 1.18.1

**Additional context**

**Comment by AndrewMac** [comment]
Actually, I've just also noticed that I'm now getting this message when updating with Composer:

`Generating optimized autoload files
Class verbb\formie\events\ModifyFieldValueForIntegrationEvent located in ./vendor/verbb/formie/src/events/ModifyFieldConfigEvent2.php does not comply with psr-4 autoloading standard. Skipping.`

This must have come up quite recently, I think, as I had no psr-4 issues a while back.

Any chance this might be at least part of the issue?

**Comment by engram-design** [comment]
Not sure how that one slipped past! Should be fixed in [1.3.16.1](https://github.com/verbb/formie/releases/tag/1.3.16.1)

**Comment by AndrewMac** [comment]
Thanks so much. Looks like it's all back to normal now! :-)
